build.gradle

```diff
@@ -1,83 +1,61 @@
 sourceCompatibility = '11'
 
-configurations {
-    compileOnly {
-        extendsFrom annotationProcessor
-    }
-}
-
 repositories {
     mavenCentral()
+    maven { url 'https://jitpack.io' }
 }
 
-task copyLombok(type: Copy) {
-	outputs.upToDateWhen { false }
-	into "${buildDir}/libs"
-	include "lombok*.jar"
-	from configurations.compileClasspath
-}
-
-build.dependsOn(copyLombok)
-
-repositories {
-	mavenCentral()
-    maven { url 'https://jitpack.io' }
+ext {
+    springCloudVersion = "Hoxton.SR12"
 }
 
 dependencies {
-    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.12.5'
-    implementation group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.12.5'
-    implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.12.5'
-
-    implementation('org.springframework.boot:spring-boot-starter-web') {
+    implementation('org.springframework.boot:spring-boot-starter-web:2.5.5') {
         exclude(module: 'spring-boot-starter-tomcat')
     }
-    implementation('org.springframework.boot:spring-boot-starter-undertow')
-    implementation('net.logstash.logback:logstash-logback-encoder:5.3')
-    implementation('org.springframework.boot:spring-boot-starter-json')
-    implementation('org.springframework.boot:spring-boot-starter-actuator')
-    implementation('org.springframework.retry:spring-retry')
-    implementation('org.springframework:spring-aspects')
 
+    implementation 'org.springframework.boot:spring-boot-starter-undertow:2.5.5'
+    implementation 'org.springframework.boot:spring-boot-starter-actuator:2.5.5'
+    implementation 'org.springframework.cloud:spring-cloud-aws-messaging:2.2.6.RELEASE'
+
+    implementation 'net.logstash.logback:logstash-logback-encoder:6.6'
 
     // Override the default version of elasticSearch, we're still using V6
     def elasticSearchVersion = '6.8.3'
     implementation("org.elasticsearch:elasticsearch:${elasticSearchVersion}")
     implementation("org.elasticsearch.client:elasticsearch-rest-client:${elasticSearchVersion}")
-    implementation("org.elasticsearch:elasticsearch-x-content:${elasticSearchVersion}")
-    implementation("org.elasticsearch:elasticsearch-cli:${elasticSearchVersion}")
     implementation("org.elasticsearch:elasticsearch-core:${elasticSearchVersion}")
     implementation("org.elasticsearch:elasticsearch-secure-sm:${elasticSearchVersion}")
 
-    implementation('org.apache.camel:camel-spring-boot:2.24.0')
-    implementation('org.apache.camel:camel-jackson:2.24.0')
-    implementation('org.apache.camel:camel-aws:2.24.0')
-    implementation('org.apache.httpcomponents:httpmime:4.5.6')
-    implementation('com.amazonaws:aws-java-sdk:1.11.553')
-    implementation('javax.xml.bind:jaxb-api:2.3.0')
-
     implementation('org.elasticsearch.client:elasticsearch-rest-high-level-client:6.6.2')
     implementation 'com.github.awslabs:aws-request-signing-apache-interceptor:b3772780da'
```

Comment:
More for information, is this used?

Reply:
I think so, it's in the elastic search client, i'm not clear if we could use (or if there is) an official client that would take care of those internals for us. I think our elastic search instance predates the 'openSearch' fork

Reply:
It appears that Amazon have some resources on it: https://docs.aws.amazon.com/opensearch-service/latest/developerguide/what-is.html
Also looks like spring provides something too: https://spring.io/projects/spring-data-elasticsearch